hack/jenkins/e2e.sh

```diff
@@ -164,6 +166,7 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
+    "GCE\sL7\sLoadBalancer\sController"
```

Comment:
We don't currently run tests in parallel with themselves, anywhere, so don't worry about that (although I still don't quite understand why they could not logically run in parallel with themselves, provided that they run in their own namespace, which they do).

As for quota, yes, requesting additional quota for the relevant GCE projects is the right way to do (and, if you like, an explicit comment in this config, explaining that these tests are only disabled due to current lack of quota, rather than any real reason why they can't run in parallel) would be fine as a temporary solution until the quota increase goes though.